src/apis/subscriber.ts

```diff
@@ -3,8 +3,50 @@ import api from '@/utils/api';
 export const getSubscribers = async () => {
   const response = await api.get('/api/subscriber');
   const data = await response.data;
-  {
-    console.log(data, '-->checking subscriber data');
-  }
   return data;
 };
+
+export const createSubscriber = async (subscriber: any) => {
```

Comment:
shall we define a Subscriber type so that this becomes?
```suggestion
export const createSubscriber = async (subscriber: Subscriber) => {
```